### ISSUE TITLE
Update example config to appropriate timestamp types

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/osm2pgsql#1785]
While reviewing #1784 I noticed that the attributes.lua example uses `sql_type = 'timestamp'`.

`timestamp` (without time zone) [should not be used to store UTC times](https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_timestamp_.28without_time_zone.29_to_store_UTC_times). Instead, it should be `timestamptz`

**Comment by joto** [comment]
I disagree. When I store the timestamp with timezone it will show up in my local time zone when viewing the data. This is especially annoying when looking at data spanning around the daylight savings time switch. And the timestamps look different from how they are in the input file.

**Comment by mmd-osm** [comment]
FYI: We had a fairly lengthy discussion on the topic a few years back: https://github.com/openstreetmap/openstreetmap-website/issues/375

tl;dr: as soon as your local machine is not running on utc+0, reading from timestamptz columns always needs an additional `at time zone 'UTC'`, or timestamps would be converted to your local tz (which tends to be a bit annoying when working with osm data).
